.github/workflows/pull-request.yml

```diff
@@ -49,7 +49,7 @@ jobs:
       uses: ./create-environment
       id: create-environment
       env:
-        PP_ENV_NAME: ${{ format('pr-actions-{0}', runner.os) }}
+        PP_ENV_NAME: ${{ format('PP-Actions-{0}-PR-{1}', runner.os, github.event.number) }}
```

Comment:
pls change this back to `pp-actions`, it designates the repo...

Reply:
chatted offline, should be `pp-actions`, since pr already appears in the post fix